components/AvalancheForecastZoneMap.tsx

```diff
@@ -92,13 +102,112 @@ export const CARD_WHITESPACE = (1 - CARD_WIDTH) / 2; // proportion of overall wi
 export const CARD_SPACING = CARD_WHITESPACE / 2; // proportion of overall width that the spacing between two cards takes up
 export const CARD_MARGIN = CARD_SPACING / 2; // proportion of overall width that each card needs as a margin
 
+enum AnimatedDrawerState {
```

Comment:
super-nit: 

```js
const AnimatedDrawerState = {
  Hidden = 'Hidden',
  Docked = 'Docked',
  Visible = 'Visible',
} as const;
type AnimatedDrawerState = typeof AnimatedDrawerState[keyof typeof AnimatedDrawerState];
```

?

Reply:
can you point me to the reading on why enums are bad? I don't think I've seen this alternative (but I'm not a super TS expert, either).